create_project.rb

```diff
@@ -0,0 +1,26 @@
+module CreateProject
+
+  def create_project
+    @driver.navigate.to 'http://demo.redmine.org'
+    @driver.find_element(:class, 'register').click
+
+    @wait.until {@driver.find_element(:id, 'user_login').displayed?}
+
+    @login = 'login'+rand(99999).to_s
+    @password = 'Qwerty11'
+    @driver.find_element(:id, 'user_login').send_keys(@login)
+    @driver.find_element(:id, 'user_password').send_keys(@password)
+    @driver.find_element(:id, 'user_password_confirmation').send_keys(@password)
+    @driver.find_element(:id, 'user_firstname').send_keys('user_firstname')
+    @driver.find_element(:id, 'user_lastname').send_keys('user_lastname')
+    @driver.find_element(:id, 'user_mail').send_keys(@login + '@q.com')
+    @driver.find_element(:name, 'commit').click
+
+    @driver.find_element(:css, ".projects").click
+    @driver.find_element(:css, ".icon.icon-add").click
```

Comment:
in css we're able to use only one class name if it will provide needed level of uniqueness